docs/source/worked-examples-openstack.rst

```diff
@@ -0,0 +1,112 @@
+Worked Examples: Using Calico-based OpenStack
+=============================================
+
+Once you've installed OpenStack (see :doc:`openstack`) and configured its
+networking (see :doc:`opens-external-conn`) you'll want to let your users start
+playing with your brand new cloud.
+
+Before you do, it will be helpful to have a few worked examples for common
+deployment scenarios. In particular, this will make it easy for you to set up
+topologies and examine their connectivity to try to get an understanding of the
+way Calico networks behave.
+
+This document contains several sections, each of which demonstrates a
+representative deployment model for a kind of workload. The early examples are
+simple, containing small numbers of machines, scaling to the later examples
+that demonstrate quite complex arrangements. Each example contains diagrams
+that represent a connectivity graph, to add clarity to the textual
+descriptions.
+
+Example 1: Development Machine
+------------------------------
+
+In this example, a user wants to spin up a machine to use as a Linux
+development environment. This user is relatively simple: they want a GUI and
+SSH access, but relatively little else.
+
+This user is provisioned with a single OpenStack user and single OpenStack
+tenant. Neutron will automatically provision them with a single security group,
+``default``, that contains the following rules:
+
+- allow all inbound traffic from machines in the ``default`` security group
+- allow all outbound traffic to anywhere
+
+Per the instructions in :doc:`opens-external-conn`, this user cannot create
+Neutron networks or subnets, but they do have access to the networks created
+by the administrator: ``external`` and ``internal``.
+
+Because the user wants to be able to reach the machine from their own laptop,
+they need the machine to be reachable from outside the data center. In
+vanilla Neutron, this would mean provisioning it with a floating IP, but in
+Calico they instead want to make sure the VM is attached to the ``external``
+network. The ``external`` network has the UUID
+``8d5dec25-a6aa-4e18-8706-a51637a428c2``.
```

Comment:
How would they find this out?  Are they just told by the admins, or can they query it somewhere?